WHITEPAPER.md

```diff
@@ -136,49 +179,70 @@ This dynamic interest rate model responds more efficiently to changing cost of c
 
 In the event a borrower’s outstanding liabilities to a given pool exceed the borrowing capacity of their collateral in that pool, the borrower can be liquidated. Liquidations are performed to reduce the chance a borrower defaults on their loan, causing a permanent loss of lender funds.
 
-Blend uses gentle dutch auctions to liquidate borrowers without overpenalizing them or exposing them to oracle risk. The auctions are carried out in three steps:
+Blend uses gentle dutch auctions to liquidate borrowers without over-penalizing them or exposing them to oracle risk.
 
 #### Auction Initiation
 
-Any user can initiate a liquidation auction on an account that has exceeded its borrow capacity for a pool. Liquidation initiators choose user collateral to auction off, and user liabilities to and amounts involved in the liquidation.
+Any user can initiate a liquidation auction on an account that has exceeded its borrow capacity for a pool. Liquidation initiators choose user collateral and liabilities involved in the liquidation. The auction will sell the user collateral (the lot of the auction) in return for the user liabilities (the bid of the auction).
 
-However, the protocol imposes the following requirements on the liquidator:
+However, the protocol imposes the following requirements on the liquidation creator:
 
-1. The value of liabilities repaid must be less than a calculated _Target Liquidation Amount_
+1. The value of liabilities repaid must be less than a calculated _Target Liquidation Amount_ to prevent user's from getting over-liquidated.
 
-2. If all user liabilities are not being repaid, the value of all liabilities repaid must be greater than 90% of the _Target Liquidation Amount_
+2. If all user liabilities are not being repaid, the value of all liabilities repaid must be greater than 90% of the _Target Liquidation Amount_ to increase the likelihood the liquidation removes all excessively borrowed funds.
 
-**Target Liquidation Amount Formula:**
+3. The value of all collateral sold must be within 1x to 2x _Target Liquidation Amount_ unless all of the user's collateral is being sold to ensure the gradual dutch auction mechanism can find a valid price.
```

Comment:
oops, forgot to add this bullet from the doc. It's now 125%-250% btw

https://docs.google.com/document/d/1AC57-sI8MVtwoSzyvHwUvFhc48nepIBA4PTkuLLNgf8/edit#

Reply:
👍